integration-tests/test/rpc.spec.ts

```diff
@@ -426,4 +426,15 @@ describe('Basic RPC tests', () => {
         .reverted
     })
   })
+
+  describe('rollup_gasPrices', () => {
+    it('should return the L1 and L2 gas prices', async () => {
+      const result = await provider.send('rollup_gasPrices', []);
+      const l1GasPrice = await env.l1Wallet.provider.getGasPrice()
+      const l2GasPrice = await env.gasPriceOracle.gasPrice()
```

Comment:
Any chance this could result in a flaky test if the gas price oracle is updated immediately after the call to `rollup_gasPrices`?

Reply:
yeah this test will be a little sketchy against kovan

Reply:
but I can't think of a good alternative to test. Probably best to just skip this test on Kovan

Reply:
We can add them all to a `Promise.all` to slightly decrease the chance of race condition